firefox-ios/Client/Coordinators/Router/RouteBuilder.swift

```diff
@@ -156,9 +156,19 @@ final class RouteBuilder {
             return .search(url: nil, isPrivate: false)
         }
 
+        var isBrowsingActivity: Bool {
+            if featureFlags.isFeatureEnabled(.universalLinks, checking: .buildOnly) {
+                userActivity.activityType == NSUserActivityTypeBrowsingWeb ||
+                userActivity.activityType == browsingActivityType
+            } else {
+                userActivity.activityType == browsingActivityType
```

Comment:
If the feature flag is not enabled I still am checking the activity type is `browsingActivityType` since that is the activity type we are setting for our handoff actions. If I were sticking with the original logic I would just return true here, I just thought this might be a little more accurate?

Reply:
Nevermind it looks like some other tests are testing `NSUserActivityTypeBrowsingWeb` so maybe it can already be more than just the `browsingActivityType`: `UserActivityRouteTests.testDeepLinkUserActivity`